ibis/expr/types/relations.py

```diff
@@ -908,31 +909,49 @@ def fillna(
         >>> import ibis
         >>> import ibis.expr.datatypes as dt
         >>> t = ibis.table([('a', 'int64'), ('b', 'string')])
-        >>> t = t.fillna(0.0)  # Replace nulls in all columns with 0.0
-        >>> t.fillna({c: 0.0 for c, t in t.schema().items() if t == dt.float64})
-        r0 := UnboundTable[unbound_table_...]
-          a int64
-          b string
-        r1 := FillNa[r0]
-          replacements:
-            0.0
-        FillNa[r1]
-          replacements:
-            frozendict({})
+        >>> res = t.fillna(0.0)  # Replace nulls in all columns with 0.0
+        >>> res = t.fillna({c: 0.0 for c, t in t.schema().items() if t == dt.float64})
 
         Returns
         -------
         Table
             Table expression
-        """  # noqa: E501
+        """
+
+        def compatible(val_type, col_type):
```

Comment:
Should we use `rlz.castable` instead of defining this?

Reply:
Actually, `dt.castable` is what you want here.

Reply:
Thanks, fixed.